transform/transformers/PCKTransformer.py

```diff
@@ -39,8 +41,7 @@ def get_form_questions(self):
 
     def get_cs_form_id(self):
         instrument_id = self.response['collection']['instrument_id']
-
-        return self.form_ids[instrument_id]
+        return self.form_ids[self.survey['survey_id']][instrument_id]
```

Comment:
Waiting for a review from @tundish before these changes are ok to merge. He knows the implications of changing this better than I do.